requirements/static/ci/common.in

```diff
@@ -17,17 +17,20 @@ docker
 gitpython>=2.1.15
 jmespath
 jsonschema
-junos-eznc==2.4.0; sys_platform != 'win32'
+junos-eznc==2.4.0; sys_platform != 'win32' and python_version <= '3.9'
+junos-eznc==2.6.0; sys_platform != 'win32' and python_version >= '3.10'
 jxmlease; sys_platform != 'win32'
 kazoo; sys_platform != 'win32' and sys_platform != 'darwin'
 keyring==5.7.1
 kubernetes<4.0
 libnacl>=1.7.1; sys_platform != 'win32' and sys_platform != 'darwin'
-moto>=1.3.14
+moto>=1.3.14 ; python_version <= '3.9'
+moto>=2.0.0 ; python_version >= '3.10'
 napalm; sys_platform != 'win32' and python_version > '3.6'
 paramiko>=2.1.6; sys_platform != 'win32' and sys_platform != 'darwin'
 passlib>=1.7.2; sys_platform != 'win32'
-pycurl>=7.43.0.5
+pycurl>=7.43.0.5; python_version <= '3.9'
+pycurl==7.44.1; python_version >= '3.10'
```

Comment:
I'm not sure this will actually work on Windows. Pycurl used to supply a pre-compiled wheel file for Windows. The last version to do that was 7.43.0.5. If we need a newer version, we may need to start compiling our own whl files.

Reply:
It's still 7.43.0.5 for python3.9 and less, we bumped it to 7.44.1 only for python3.10